101-notification-hubs/azuredeploy.json

```diff
@@ -0,0 +1,56 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "notificationHubNamespace": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of the Notification Hub Namespace"
+            }
+        },
+        "notificationHubName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of the NotificationHub"
+            }
+        }
+    },
+    "variables": {
+        "location": "[resourceGroup().location]",
+        "nhVersion": "[providers('Microsoft.NotificationHubs', 'namespaces').apiVersions[0]]",
```

Comment:
Neat trick! However is there a possibility that this template will break automatically if a breaking change is introduced to a future api version?